dub.json

```diff
@@ -36,6 +36,7 @@
         "source/scpp/build/uint128_t.o"
     ],
     "lflags": [ "-lsodium", "-lstdc++", "-lsqlite3" ],
+    "buildRequirements": [ "allowWarnings" ],
```

Comment:
Would it be possible to make this versioned, e.g. only use this flag for the `ddox` target?

Reply:
Not that I know of. But while they're allowed, we'll still see them on the console.